log/auth.go

```diff
@@ -1,28 +0,0 @@
-package log
-
-type eventAuth struct {
-	Identity     string       `json:"identity,omitempty"`
-	IdentityType identityType `json:"identity_type,omitempty"`
-}
-
-type identityType string
-
-const (
-	// SERVICE represents a service account type
-	SERVICE identityType = "service"
-	// USER represents a user account type
-	USER identityType = "user"
-)
-
-func (l *eventAuth) attach(le *EventData) {
-	le.Auth = l
-}
-
-// Auth returns an option you can pass to Event to include identity information,
-// for example the identity type and user/service ID from an inbound HTTP request
-func Auth(identityType identityType, identity string) option {
-	return &eventAuth{
-		Identity:     identity,
-		IdentityType: identityType,
-	}
-}
```

Comment:
This not used?

Reply:
Nope, literally not used anywhere so no point reimplementing in v3 hence I removed it.
https://github.com/search?q=org%3AONSdigital+.Auth%28++language%3AGo&type=code&l=Go